### PR TITLE
boxes 2.2.0

### PR DESCRIPTION
**Pull request by Porkepix**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/boxes.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 class Boxes < Formula
   desc "Draw boxes around text"
   homepage "https://boxes.thomasjensen.com/"
-  url "https://github.com/ascii-boxes/boxes/archive/v2.1.1.tar.gz"
-  sha256 "95ae6b46e057a79c6414b8c0b5b561c3e9d886ab8123a4085d256edccce625f9"
-  license "GPL-2.0-only"
+  url "https://github.com/ascii-boxes/boxes/archive/v2.2.0.tar.gz"
+  sha256 "98b8e3cf5008f46f096d5775d129c34db9f718728bffb0f5d67ae89bb494102e"
+  license "GPL-3.0"
```
Comment (carlocab) [inline Formula/boxes.rb:6]: `-only` or `-or-later`?

File: Formula/boxes.rb [hunk 2]
```diff
@@ -1,9 +1,9 @@
 class Boxes < Formula
   desc "Draw boxes around text"
   homepage "https://boxes.thomasjensen.com/"
-  url "https://github.com/ascii-boxes/boxes/archive/v2.1.1.tar.gz"
-  sha256 "95ae6b46e057a79c6414b8c0b5b561c3e9d886ab8123a4085d256edccce625f9"
-  license "GPL-2.0-only"
+  url "https://github.com/ascii-boxes/boxes/archive/v2.2.0.tar.gz"
```
Comment (carlocab) [inline Formula/boxes.rb:4]: Not the latest GitHub release.

File: Formula/boxes.rb (hunk 1 above)
Comment (Porkepix) [reply Formula/boxes.rb:6]: I picked what the CI gave, how is the difference done and which one should be picked when the license file just give GPL v3?

Comment (SMillerDev) [reply Formula/boxes.rb:6]: Check the source files, they should indicate the restrictions on the license

Comment (Porkepix) [reply Formula/boxes.rb:6]: I don't see any `later` so I guess that means it's only? https://github.com/ascii-boxes/boxes/blob/master/LICENSE

Comment (carlocab) [reply Formula/boxes.rb]: Yea, seems like `-only`. https://github.com/ascii-boxes/boxes/blob/d572344fced538d307d8f4eafaafd5b8133ce003/src/boxes.c#L1-L14

Comment (carlocab) [reply Formula/boxes.rb]: See also https://github.com/ascii-boxes/boxes/issues/101#issuecomment-1250377136

**Comment by mathomp4** [comment]
> /cc @mathomp4

I am summoned. How can I help?

**Comment by tsjensen** [comment]
> > /cc @mathomp4
> 
> I am summoned. How can I help?

I just wanted to make you aware of this going on, as I stumbled across it and didn't see that you were involved yet. Nothing to do, as far as I am concerned! 😄

**Comment by mathomp4** [comment]
> > > /cc @mathomp4
> > 
> > 
> > I am summoned. How can I help?
> 
> I just wanted to make you aware of this going on, as I stumbled across it and didn't see that you were involved yet. Nothing to do, as far as I am concerned! 😄

Ah okay. I did see boxes 2.2 was out and have been waiting for it to appear with `brew upgrade`! Time to cowsay so much 😄

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**